### ISSUE TITLE
Add an option for a terminal only dock or lock a dock

### ISSUE THREAD

**Issue by gawlk** [zed-industries/community#995]
### Check for existing issues

- [X] Completed

### Describe the feature

Hi,

Love the work you're doing with the editor but there is one little thing that's really annoying.

My workflow is usually:

| File tree | Main pane with all files | Terminal dock |

But when I focus on the terminal to do something and then open a file it'll open it in dock where the terminal is.

I would've assumed it would open it in the main pane/dock where all the files are like it's the case in VS Code.

**Comment by JosephTLyons** [comment]
Hey @gawlk, we actually just landed a change to this in our preview build (v0.75.0) that forces all new items to open up in the center pane - nothing will be opened up in the dock.  We know that users will still want to move items to the dock, so we added a command to make it super easy to bring a focused item from the center pane -> dock, and from dock -> center pane.

This change should go out in the next week or so.

Here's the original issue:

- https://github.com/zed-industries/community/issues/533

Closing this one out.  Appreciate the feedback, keep it coming :).
